misc/profiles2/trekking.brf

```diff
@@ -90,22 +231,26 @@ assign initialcost =
 # implicit access here just from the motorroad tag
 # (implicit access rules from highway tag handled elsewhere)
 #
-assign defaultaccess =
-       if access= then not motorroad=yes
-       else if access=private|no then false
-       else true
+assign defaultaccess = not ( access=private|no )
```

Comment:
This (and below) is a fix for https://github.com/nrenner/brouter-web/issues/31. This new behavior can be disabled by a variable at the beginning on the profile.